### DISCUSSION TITLE
Share your yearly standup - Community Call

### DISCUSSION THREAD

**Discussion by LuffySama-Dev** [EddieHubCommunity/support#3007] [Community Call]
**2021 standup**
- what we achieved this year
- what we could have done better
- what we plan to achieve next year"

**Comment by suitupstan** [comment]
- I've achieved nothing this year yet. I might get it when I completed my 1.5k contribs (100 to go 🚀)
- I am silly, I have done probably everything wrong 😂 So I have to fix everything
- I am planning to make more valuable contributions the next year 💪

**Reply by eddiejaoude** [reply]
Thank you for sharing and support 👍

**Reply by suitupstan** [reply]
Thanks for the call 🙌

**Comment by anuraglol** [comment]
- I completed Hacktoberfest this year (thanks to EddieHub)
- Reached 200 Followers on Twitter
- Not many achievements, but looking forward to leveling it all up in 2022

**Reply by eddiejaoude** [reply]
Congrats on completing hackatoberfest 🎉

**Reply by suitupstan** [reply]
200 followers sound awesome 🎊 btw I also completed my first hacktoberfest this year! 🥳

**Reply by anuraglol** [reply]
Cool! Thanks :yeye:!

**Comment by LuffySama-Dev** [comment]
1.  I started to contribute to open source. 
2.  I could have started earlier with Opensource.
3.  I haven't achieved much, looking forward to contribute more in openSource.

**Reply by eddiejaoude** [reply]
Yay contributing to open source 🎉

**Reply by suitupstan** [reply]
All Open Source ❤️

**Comment by Rohit-kumar-yadav** [comment]
• completed my first Hacktoberfest
• Everything lot of thing done wrong
• Planning to learn more tech stack and also some valuable contribution😊

**Reply by eddiejaoude** [reply]
Nice work on completing hackatoberfest 🎉

**Reply by suitupstan** [reply]
You are doing all great, Nothing can go wrong on GitHub ;) Contributions & Hacktoberfest 🔥

**Comment by Shivansh-yadav13** [comment]
- Joined EddieHub Community 😁
- Completed my first Hacktoberfest
- Won my first hackathon
- Involvement in Tech
- Started Open Source
- Planning to do open source more

**Reply by eddiejaoude** [reply]
Yay joined EddieHub! Congrats on winning the hackathon 🎉

**Reply by suitupstan** [reply]
Congrats on all of those achievements! 🎊

**Comment by avneesh0612** [comment]
- I learnt how to code, did technical writing, contributed to open source. I also started to freelance and won a hackathon!
- I wish I had started contributing to open source earlier 🙄
- I am hoping to contribute to 50 projects in 2022, publish more content.

**Reply by eddiejaoude** [reply]
great year! open source, freelancing and won a hackathon 🎉

**Reply by avneesh0612** [reply]
Thanks 🙌

**Reply by suitupstan** [reply]
I love the 50+ repo contribution 🎊

**Comment by ewainy** [comment]
- What I achieved this year? Got my first tech job! 🥳 
- What could I have improved? Sharing my learning journey from the start and learning how to contribute to open source
- What I plan to achieve next year? I want to give back more to the community with open source and also help people get their first tech job 🚀

**Reply by eddiejaoude** [reply]
amazing! I can't wait to hear more about your first tech job 🔥

**Reply by suitupstan** [reply]
Awesome! Do share your experience!
